client/index.jsx

```diff
@@ -1,8 +1,51 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import App from './app';
+import React, { useRef } from 'react';
+import reactDOM from 'react-dom/client';
 
 const container = document.querySelector('#root');
-const root = ReactDOM.createRoot(container);
+const root = reactDOM.createRoot(container);
 
-root.render(<App />);
+function MusicWidget() {
+  const widgetRef = useRef(null);
+  return (
+    <div
+      className="music-widget"
+      ref={widgetRef}
+      style={{ width: '400px', height: '200px', background: '#93B1A7' }}
+      onMouseDown={e => {
+        e.preventDefault();
+        widgetRef.current.style.position = 'absolute';
+        const shiftX = e.clientX - widgetRef.current.getBoundingClientRect().left;
+        const shiftY = e.clientY - widgetRef.current.getBoundingClientRect().top;
+
+        const onMouseMove = e => {
```

Comment:
Uggh! We need to change that template's lint config to use `()` around arrow function parameters!